### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_AzureNotebooksWelcome_Inspect: Label is missing for combobox (i.e. Markdown) on page.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#3938]
**Environment Details:**
Application Name: AzureNotebooks_Jupyter
Login URL: notebooks.azure.com
Browser Details:
Edge: Microsoft Edge 42.17134.1.0
Chrome: Version 68.0.3440.106
Operating System:Windows10

**Repro Steps:**
1. Login with the appropriate credentials to azure notebook account.
2. Navigate to run button & hit enter --> Navigate to Azure Notebooks welcome & hit enter.
3. Verify label relationship of controls on page(i.e. for markdown combobox).

**Actual:**
Label is missing for combobox (i.e. Markdown) form element on page.
Same issue observed on all pages where these combobox is available.

**Expected:**
Form element or combobox should have meaningful label on page.

**User Impact:**
Screen reader user would not get information about what type of combobox present on page.

**Recommendation:**
Fix any of the following:
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty or not visible

**Attachment:**
![a11y_azurenotebooks_jupyter_azurenotebookswelcome_inspect_label](https://user-images.githubusercontent.com/39204909/45752534-6cd43280-bc33-11e8-81dc-1b4709c91e80.jpg)

**Comment by takluyver** [comment]
Closed by #4685.
